src/client.ts

```diff
@@ -21,14 +21,84 @@ import {
 } from './http';
 import { generateRandomString, getTimeInSeconds } from './util';
 
-export type ClientOptions = {
+export type RequiredClientOptions = {
```

Comment:
Looks like name itself should not introduce any backwards incompatibility. Since there should be no reference on type name from usage. 

But if previous code use `const duo_client = Client(id, secret.....)`, it need to add more option. But I guess it is a drawback coming with builder?

Reply:
Is there any way to avoid that problem?  could we make one of the parameters optional?  defaulted?